MapboxNavigation/NavigationMapView.swift

```diff
@@ -87,6 +87,8 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
     var animatesUserLocation: Bool = false
     var altitude: CLLocationDistance = defaultAltitude
     var routes: [Route]?
+    var altRouteSourceIdentifiers: [String] = []
```

Comment:
We only need one source for all the routes. The source’s shape would be an MGLShapeCollectionFeature containing one MGLPolylineFeature for each route. Each polyline feature’s identifier would be the index of that route.